superset/db_engine_specs/base.py

```diff
@@ -217,7 +225,7 @@ def make_select_compatible(
         return select_exprs
 
     @classmethod
-    def fetch_data(cls, cursor, limit: int) -> List[Tuple]:
+    def fetch_data(cls, cursor: Any, limit: int) -> List[Tuple]:
```

Comment:
The DBAPI cursor isn't wrapped by SQLALchemy and thus for now I used a generic type.